package.json

```diff
@@ -19,24 +19,25 @@
     "color-string": "^1.5.3",
     "dompurify": "2.0.17",
     "dotenv": "^8.2.0",
-    "gatsby": "2.24.85",
-    "gatsby-plugin-canonical-urls": "^2.2.3",
-    "gatsby-plugin-catch-links": "^2.2.1",
-    "gatsby-plugin-manifest": "^2.4.0",
-    "gatsby-plugin-mdx": "^1.2.40",
+    "gatsby": "3.1.2",
+    "gatsby-plugin-canonical-urls": "^3.1.0",
+    "gatsby-plugin-catch-links": "^3.1.0",
+    "gatsby-plugin-dark-mode": "^1.1.2",
+    "gatsby-plugin-manifest": "^3.1.0",
+    "gatsby-plugin-mdx": "^2.1.0",
     "gatsby-plugin-meta-redirect": "^1.1.1",
-    "gatsby-plugin-offline": "^3.1.2",
-    "gatsby-plugin-react-helmet": "^3.2.4",
-    "gatsby-plugin-sass": "2.3.1",
-    "gatsby-plugin-sharp": "2.6.36",
-    "gatsby-plugin-sitemap": "^2.0.5",
-    "gatsby-plugin-typescript": "^2.3.5",
-    "gatsby-remark-autolink-headers": "^2.2.1",
-    "gatsby-remark-copy-linked-files": "^2.10.0",
-    "gatsby-remark-images": "^3.2.2",
-    "gatsby-remark-prismjs": "^3.5.0",
-    "gatsby-source-filesystem": "^2.2.5",
-    "gatsby-transformer-remark": "^2.8.5",
+    "gatsby-plugin-offline": "^4.1.0",
+    "gatsby-plugin-react-helmet": "^4.1.0",
+    "gatsby-plugin-sass": "4.1.0",
+    "gatsby-plugin-sharp": "3.1.2",
+    "gatsby-plugin-sitemap": "^3.1.0",
+    "gatsby-plugin-typescript": "^3.1.0",
+    "gatsby-remark-autolink-headers": "^3.1.0",
+    "gatsby-remark-copy-linked-files": "^3.1.0",
+    "gatsby-remark-images": "^4.1.1",
+    "gatsby-remark-prismjs": "^4.1.0",
+    "gatsby-source-filesystem": "^3.1.0",
+    "gatsby-transformer-remark": "^3.1.0",
```

Comment:
@benhalverson Is it okay to upgrade the versions in this PR?

Reply:
Not yet. Another issue still exists with webpack 5 gatsby and storybook. See https://github.com/nodejs/nodejs.dev/pull/1178

Reply:
Okay ill revert the version. Thanks for letting me know!